server/command.go

```diff
@@ -150,6 +150,7 @@ func addSubCommands(jira *model.AutocompleteData, optInstance bool) {
 	jira.AddCommand(createWebhookCommand(optInstance))
 
 	// Help and info
+	jira.AddCommand(model.NewAutocompleteData("setup", "", "Start Jira plugin setup flow"))
```

Comment:
Upon taking another look at this change, I realized we should make it so this command is shown to system admins. We can do this with this change:

```suggestion
	setup := jira.AddCommand(model.NewAutocompleteData("setup", "", "Start Jira plugin setup flow"))
	setup.RoleID = model.SystemAdminRoleId
```